### PR TITLE
Add payment gateways section in Integrations section

### PR DESCRIPTION
**Pull request by MonikaMrozek**
pmt gateways description moved from Saleor Server > Guides > How to Integrate New Pmt Gateway  to Saleor Server > Integrations > Supported Payment Gateways.

Plugins topic modified for Stripe activation.

### CODE REVIEW

File: docs/dashboard/configuration/plugins.md [hunk 1]
```diff
@@ -9,19 +9,29 @@ Saleor provides the possibility to add various extensions to it so you can creat
...
 
+* Used to process payments (all supported payment gateways in Saleor come as plugins):
+
+    * Braintree
+
+    * Razorpay
+
+    * Stripe
+
+You need to provide public and private keys to activate each payment gateway.
```
Comment (salwator) [inline docs/dashboard/configuration/plugins.md]: Each gateway might need additional configuration to activate, ex. merchant id on _braintree_.

File: docs/integrations/payment-gateways.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+> **Note**
+>
+> All payment backends default to using sandbox mode. This is very useful for development. When deploying to a production server, make sure you use the production mode.
+
+### Braintree 
+
+This gateway implements payments using [Braintree](https://www.braintreepayments.com/).
+
+| Environment Variable | Description |
```
Comment (salwator) [inline docs/integrations/payment-gateways.md]: We're no longer using environment variables, should refer to configuration options in plugin instead.

File: docs/integrations/payment-gateways.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+| --- | --- |
+| `BRAINTREE_SANDBOX_MODE` | Whether to use a sandbox environment for testing, `True` (default) or `False`. |
```
Comment (salwator) [inline docs/integrations/payment-gateways.md]: Same.

File: docs/integrations/payment-gateways.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+>
+> Make sure that Braintree’s currency is the same as your shop’s currency, otherwise customers will be charged the wrong amount.
+
+### Razorpay
+
+This gateway implements payments using [Razorpay](https://razorpay.com/).
+
+Before you start, you need to create your API credentials. Go to your Razorpay account settings, and to [the API Keys section](https://dashboard.razorpay.com/#/app/keys).
+
+| Environment Variable | Description |
```
Comment (salwator) [inline docs/integrations/payment-gateways.md]: Same

File: docs/integrations/payment-gateways.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+> **Warning**
+>
+> Currently Razorpay only supports the paisa (INR) currency.
+
+### Stripe
+
+This gateway implements payments using [Stripe](https://stripe.com/).
+
+| Environment Variable | Description |
+| --- | --- |
```
Comment (salwator) [inline docs/integrations/payment-gateways.md]: Same, plus here we no longer use most of this, like store name, store image, prefill, remember_me, locale, enable_*

File: docs/integrations/payment-gateways.md [hunk 6]
```diff
@@ -43,13 +44,13 @@ This gateway implements payments using [Razorpay](https://razorpay.com/).
...
+| Plugin Settings | Description |
 | --- | --- |
-| `RAZORPAY_PUBLIC_KEY` | Your Razorpay key id. |
-| `RAZORPAY_SECRET_KEY` | Your Razorpay secret key id. |
-| `RAZORPAY_PREFILL` | Pre-fill the email and customer’s full name if set to `True` (default). |
-| `RAZORPAY_STORE_NAME` | Your store name. |
-| `RAZORPAY_STORE_IMAGE` | An absolute or relative link to your store logo. |
+| `Public API key` | Your Razorpay key id. |
+| `Secret API key` | Your Razorpay secret key id. |
+| `Store customers card` | Determines if Saleor should store cards on payments. |
```
Comment (NyanKiyoshi) [inline docs/integrations/payment-gateways.md]: This is not supported by the backend. Should we document this setting?

Comment (salwator) [reply docs/integrations/payment-gateways.md]: @NyanKiyoshi you obviously now more about Razorpay then I do, could you please see if there is a need to update Razopay plugin and its configuration? Maybe we could start with **not** documenting unwanted settings.

Comment (NyanKiyoshi) [reply docs/integrations/payment-gateways.md]: After going through their new documentation, looks like it is a usable feature. Maybe we could just add a note that it not currently possible then? That way would be fine.

Comment (salwator) [reply docs/integrations/payment-gateways.md]: @MonikaMrozek can we just add a note here with this information?

Comment (MonikaMrozek) [reply docs/integrations/payment-gateways.md]: Not sure what the note should read, guys? What is not currently possible?

**Pull request merged**